package.json

```diff
@@ -105,8 +106,11 @@
     "@botpress/chat-skin": "@botpress/chat-skin/src"
   },
   "dependenciesMeta": {
-    "@blueprintjs/core@3.52.0": {
-      "unplugged": true
+    "core-js@3.20.0": {
+      "built": false
+    },
+    "core-js-pure@3.20.0": {
+      "built": false
```

Comment:
Why not build them?

Reply:
When using PnP, this adds 7000 files to the repo, out of a total of 10000 that is produce, so it reduces the amount of files to 3000 afterwards. I haven't seen anything not working from not building those. I think storybook uses them

Reply:
Can you put this on another branch that we can then use to give PnP another go? That way, we won't pollute the repo with code/configs that are currently unused or unavailable.